psychopy/monitors/calibTools.py

```diff
@@ -498,10 +498,11 @@ def _loadAll(self):
         if not os.path.exists(thisFileName):
             self.calibNames = []
         else:
-            thisFile = open(thisFileName, 'rb')
             if ext==".json":
+                thisFile = open(thisFileName, 'r')
```

Comment:
While you're at it, would you be willing to switch to a context manager here? i.e.
```python
with open(thisFileName, 'r') as thisFile:
    ...
```
:)

Reply:
Also it would be great if you could add a test so we would catch this kind of problem "automatically" in the future, should a regression occur!

Reply:
Can do. Where do you think the test should live? tests/test_misc? Or should there be a tests/test_monitors?

Reply:
> Or should there be a tests/test_monitors?

I believe this would be best, yes. Thanks!